src/js/stores/CosmosPackagesStore.js

```diff
@@ -263,6 +280,15 @@ class CosmosPackagesStore extends GetSetBaseStore {
     return null;
   }
 
+  getPackages() {
```

Comment:
Won't this return an instance of `UniversePackage` with properties that equal `packageNames` and values that equal the versions we've fetched for those `packageNames`?

Reply:
I find it really strange it returns an single instance of `UniversePackage` with all the packages contained inside, instead of an array of `UniversePackage`s for each package.

Reply:
That was a bad choice I made. When I weighted creating a new struct or reusing what `UniversePackages` struct I decided to reuse.

I'm addressing this creating a new struct and renaming a couple things. thanks.